<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewXMLClass</class>
 <widget class="QMainWindow" name="ViewXMLClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1431</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Таблица класса</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Текущий класс:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboClass"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Предок</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelParentClass">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="curClass">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Текущий метод:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="curMethod">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Константы класса</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Поля</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Методы</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Локальные перменные</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" rowspan="2">
     <widget class="QTableWidget" name="tw_table_class_fields">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Номер</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Доступ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="2" rowspan="2">
     <widget class="QTableWidget" name="tw_table_classes_method">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Номер</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Доступ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Статичный?</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дескриптор</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="3" rowspan="2">
     <widget class="QTableWidget" name="tw_table_local_var">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Номер</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTableWidget" name="tw_table_class_const_var">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Номер</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Значение</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QTableWidget" name="tw_table_classes">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Предок</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1431</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
